### ISSUE TITLE
Fix quoting of opam arguments

### ISSUE THREAD

**Issue by samoht** [mirage/functoria#54]
See https://github.com/mirage/mirage/pull/519

**Comment by samoht** [comment]
There seem to be a bug in `depext` which doesn't support `>=` constraints anyway, so will work-around that bug by just removing the constraint...

```
     method packages =
 +        (* FIXME: fix the bug in functoria upstream *)
 +        let quote s = Printf.sprintf "'%s'" s in
          let l = [
 -          "lwt"; "mirage-types" ^ mirage_types_version; "mirage-types-lwt"
 +          "lwt"; quote ("mirage-types" ^ mirage_types_version);
 +          "mirage-types-lwt"
          ] in
          Key.(if_ is_xen) ("mirage-xen" :: l) ("mirage-unix" :: l)
```

**Comment by hannesm** [comment]
solved by #84
